tests/console/docker.pm

```diff
@@ -34,7 +34,8 @@ sub run {
 
     install_docker_when_needed;
 
-    die('boo#1072367 - docker engine doesnt have seccomp support') unless (script_output('docker info') =~ m{seccomp/});
+    # boo#1072367 - docker engine doesnt have seccomp support
+    assert_script_run('docker info | grep seccomp');
```

Comment:
I don't understand. How does this fix stuff?

hint: You can put the comment within the [fail_message](http://open.qa/api/testapi/#_assert_script_run)